### DISCUSSION TITLE
Add pomodoro timer option

### DISCUSSION THREAD

**Discussion by Zarpyk** [solidtime-io/solidtime-desktop#80] [Feature Requests]
Add pomodoro timer, something like this (GIF of [Superproductivity](https://super-productivity.com/)):
![Solidtime](https://github.com/user-attachments/assets/cd6ee55f-832e-4b7c-bba6-7bae60af634e)

**Comment by repparw** [comment]
Toggl also has an implementation, though I believe it's only on mobile?
Has a lot of extra things like background sound, auto-start, etc.
maybe worth checking
